content/en/docs/ibm/create-cluster.md

```diff
@@ -128,12 +189,150 @@ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
   --public-vlan ${PUBLIC_VLAN_ID} 
 ```
 
-Wait until the cluster is deployed and configured. It can take a while for the cluster to be ready. Run with following command to periodically check the state of your cluster. Your cluster is ready when the state is `normal`.
+Wait until the cluster is deployed and configured. It can take a while for the cluster to be ready. Run with following
+command to periodically check the state of your cluster. Your cluster is ready when the state is `normal`.
 
 ```shell
 ibmcloud ks clusters --provider ${WORKER_NODE_PROVIDER} |grep ${CLUSTER_NAME}|awk '{print "Name:"$1"\tState:"$3}'
 ```
 
+## Create an IBM Cloud Kubernetes cluster for `vpc-gen2` infrastructure
+
+Creating a `vpc-gen2` based cluster needs a VPC, a subnet and a public gateway attached to it. Fortunately, this is a one
+time setup. Future `vpc-gen2` clusters can reuse the same VPC/subnet(with attached public-gateway). 
+
+1. Begin with installing a vpc-infrastructure plugin:
+
+   `$ ibmcloud plugin install vpc-infrastructure`
```

Comment:
```suggestion
   ```shell
   ibmcloud plugin install vpc-infrastructure
   ```
```

and close the code block with ```